CMakeLists.txt

```diff
@@ -33,6 +33,12 @@ HunterGate(
 
 string(COMPARE NOTEQUAL "$ENV{TRAVIS_TAG}" "" travis_deploy)
 string(COMPARE EQUAL "$ENV{APPVEYOR_REPO_TAG}" "true" appveyor_deploy)
+string(COMPARE EQUAL "$ENV{CI}" "true" is_ci)
```

Comment:
We already have `drishti_ci`:
* https://github.com/elucideye/drishti/blob/ba065a0853550bf401d9efd69a30791c492b7a7f/CMakeLists.txt#L269-L275